<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appearanceconfig</class>
 <widget class="QWidget" name="appearanceconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>283</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="style_label">
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="style"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="rotate_label">
     <property name="text">
      <string>Rotate vertical frames:</string>
     </property>
     <property name="buddy">
      <cstring>rotate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="rotate">
     <property name="toolTip">
      <string>&lt;p&gt;Controls whether the frames drawn around taskbar entries should be rotated 90-degrees counter-clockwise when the taskbar is in a vertical panel.&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="toolTips_label">
     <property name="text">
      <string>Tooltips:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>toolTips</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="toolTips"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="previewSize_label">
     <property name="text">
      <string>Window preview size:</string>
     </property>
     <property name="buddy">
      <cstring>previewSize</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KIntSpinBox" name="previewSize">
     <property name="toolTip">
      <string>&lt;p&gt;Controls the width of window previews with tooltips.&lt;/p&gt;</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="launcherIcons_label">
     <property name="text">
      <string>Always use launcher icons:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>launcherIcons</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="launcherIcons">
     <property name="toolTip">
      <string>&lt;p&gt;Enabling this item forces the icon for a running application to be the same as that used for the launcher. This resolves some oddities where the launcher icon is different from the application icon - as happens with LibreOffice.&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="iconScale_label">
     <property name="text">
      <string>Scale icons to:</string>
     </property>
     <property name="buddy">
      <cstring>iconScale</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="KIntSpinBox" name="iconScale">
     <property name="toolTip">
      <string>&lt;p&gt;Controls the scaling of the taskbar icon. When set to &amp;quot;Automatic&amp;quot;, the taskbar will attempt to determine the optimal size.&lt;/p&gt;</string>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="spacing_label">
     <property name="text">
      <string>Spacing:</string>
     </property>
     <property name="buddy">
      <cstring>spacing</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="KIntSpinBox" name="spacing">
     <property name="toolTip">
      <string>&lt;p&gt;Set the amount of extra spacing between items.&lt;/p&gt;</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="maxRows_label">
     <property name="text">
      <string>Maximum rows:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>maxRows</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="KIntSpinBox" name="maxRows">
     <property name="toolTip">
      <string>&lt;p&gt;Controls the maximum number of rows (for a horizontal taskbar), or columns (for a vertical taskbar), that will be used.&lt;/p&gt;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="sortingStrategy_label">
     <property name="text">
      <string>Sorting:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>sortingStrategy</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="sortingStrategy">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="showSeparator_label">
     <property name="text">
      <string>Separator:</string>
     </property>
     <property name="buddy">
      <cstring>showSeparator</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="showSeparator">
     <property name="toolTip">
      <string>&lt;p&gt;When enabled, and there is only 1 row/column, then a separator will be drawn between the launchers/tasks-with-launchers and non-launcher tasks.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="highlightWindows_label">
     <property name="text">
      <string>Highlight windows:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="highlightWindows">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>style</tabstop>
  <tabstop>rotate</tabstop>
  <tabstop>toolTips</tabstop>
  <tabstop>previewSize</tabstop>
  <tabstop>launcherIcons</tabstop>
  <tabstop>iconScale</tabstop>
  <tabstop>spacing</tabstop>
  <tabstop>maxRows</tabstop>
  <tabstop>sortingStrategy</tabstop>
  <tabstop>showSeparator</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
